src/dotnet/Audio.Service/AudioProcessor.cs

```diff
@@ -84,66 +84,66 @@ public AudioProcessor(Options settings, IServiceProvider services)
                 TimeSpan.Zero,
                 AudioSourceLog,
                 cancellationToken);
-            var openSegment = new OpenAudioSegment(0, record, audio, author, languages, OpenAudioSegmentLog);
+            var openSegment = new OpenAudioSegment(0,
```

Comment:
I ended up putting this on one line at some point, coz cut something & it was fitting. Sorry :)